packages/fxa-settings/src/lib/auth-errors/auth-errors.ts

```diff
@@ -666,7 +666,7 @@ export const getLocalizedErrorMessage = (
 
 export const composeAuthUiErrorTranslationId = (err: {
   errno?: number;
-  message: string;
+  message?: string;
```

Comment:
This is because of `errno` for `AuthServerError` is possibly undefined unfortunately, and I didn't dig into that. I have a comment in `container.tsx` and can file an issue for it:

```
// TODO: why is `errno` in `AuthServerError` possibly undefined?
// might want to grab from `ERRORS.UNEXPECTED_ERROR` instead
```